### PR TITLE
[2] Implement server for front end serving

### PR DESCRIPTION
**Pull request by katevi**
1. In order to deploy front-end, you'd have to have a web server. Please, implement it and include into front-end part source code with instructions on how to run t.
I'd consider to do a light weight implementation, for example: NodeJS Express. #6 
2. Also this server must be handle case if back end server is unavailable. #13

### CODE REVIEW

File: chat-client/server.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const express = require('express')
+const app = express()
+const port = 8081
+
+const stompjs = require('stompjs');
+const sockjs = require('sockjs-client');
```
Comment (AndreyKoltsov1997) [inline chat-client/server.js:6]: Make sure to write down README.md part on how to launch the server (including installation of prerequisites, such as express, stompjs, sockjs-client).

Comment (katevi) [reply chat-client/server.js:6]: I have some problems with server.js:
I don't know, how to use functions in server.js from another file (app.js)

File: chat-client/server.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+app.get('/', (req, res) => {
+    res.sendFile(__dirname + '/app/index.html');
+	res.sendFile(__dirname + '/app/app.js');
+});
+
+
+app.listen(port, (err) => {
+    if (err) {
+        return console.log('something bad happened', err)
```
Comment (AndreyKoltsov1997) [inline chat-client/server.js:16]: I'd change the message to 
```
...
`Unable to start the server. Details: ${err}`
...
```

Also, change the level of the logged message to errors output stream`console.err(...).`
* More about output streams available via `console`: https://nodejs.org/api/console.html#console_console_error_data_args

**Comment by katevi** [comment]
Added handling unavailable server case #13:
image

**Pull request merged**